src/hooks/sort-by.js

```diff
@@ -12,18 +11,39 @@ export default class extends BaseHooks {
 
         this.sortByEvents();
     }
+    // ie 11 does not support native closest so use this function instead
+    closest(el, selector) {
```

Comment:
We should have unit tests for this function to prove that it behaves the same as https://github.com/jquery/jquery/blob/30e1bfbdcb0ff658f1fa128b72480194e8ecb926/src/traversing.js#L42